massgov/mayflower
# Automate the updating of the release PR and tag descriptions

## Description
[https://jira.mass.gov/browse/DP-17065](https://jira.mass.gov/browse/DP-17065)
change types are not consolidated, e.g. https://github.com/massgov/mayflower/releases/tag/9.37.0

### Before: 
```
### Added 
- (0) [enter a valid component name - see direction above] add a Jira ticket or issue number,e.g. DP-12345 or 124: Describe the change. If you need multiple lines, start the first line with the following "|-" characters. (#PR)
### Added 
- (1) [enter a valid component name - see direction above] add a Jira ticket or issue number,e.g. DP-12345 or 124: Describe the change. If you need multiple lines, start the first line with the following "|-" characters. (#PR)
```
### After: 
```
### Added 
- (0) [enter a valid component name - see direction above] add a Jira ticket or issue number,e.g. DP-12345 or 124: Describe the change. If you need multiple lines, start the first line with the following "|-" characters. (#PR)
- (1) [enter a valid component name - see direction above] add a Jira ticket or issue number,e.g. DP-12345 or 124: Describe the change. If you need multiple lines, start the first line with the following "|-" characters. (#PR)
```

## To test:
1. checkout 23ab19b
2. `node scripts/release-branch.js`

or see 446d4f6 for compiled changelog
